src/classes/Customer.js

```diff
@@ -1,7 +1,9 @@
 class Customer {
-  constructor() {
-
+  constructor(id, name) {
+    this.id = id;
+    this.name = name;
   }
+  
```

Comment:
I'm not sure if it's necessary to include the customer specific bookings or total spent in your constructor since you added it to your bookingsLog. But think about how you want to store customer specific bookings vs all bookings.